tests/numpy/test_sampled_from.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+
+from hypothesis import given, assume
+from hypothesis.extra import numpy as npst
+from tests.common.utils import checks_deprecated_behaviour
+from hypothesis.strategies import data, sampled_from
+
+
+@given(data(), npst.arrays(
+    dtype=npst.scalar_dtypes(),
+    shape=npst.array_shapes(max_dims=1)
+))
+def test_can_sample_numpy_array_without_warning(data, arr):
+    elem = data.draw(sampled_from(arr))
+    try:
+        assume(not np.isnan(elem))
+    except TypeError:
+        pass
+    assert elem in arr
+
+
+@checks_deprecated_behaviour
+@given(data(), npst.arrays(
+    dtype=npst.scalar_dtypes(),
+    shape=npst.array_shapes(min_dims=2, max_dims=5)
+))
+def test_multi_dimensional_arrays_are_a_no(data, arr):
```

Comment:
(minor) nit: is this the best method name for this?

Reply:
Probably not, but it works!  Are you otherwise happy with the pull?

Reply:
yeah, seems good. I see you mentioned something about the tests - is that for the tests in-repo, not on the runner side?

Reply:
I meant that having committed 92cc3242d4eb6f7faafc13b496236acdd3dccc27, I shouldn't do final review.

I've updated the test names now, so once the tests pass merge away!